__tests__/server.test.js

```diff
@@ -287,4 +287,32 @@ describe('DELETE /api/comments/:comment_id', () => {
             expect(errorMsg).toBe('404: Not Found')
         })
     })
+})
+describe('GET /api/users', () => { 
+    test('200: responds with an array of user objects with correct properties', () => {
+        return request(app)
+        .get('/api/users')
+        .expect(200)
+        .then(( { body } ) => {
+            const users = body.users
+            expect(Array.isArray(users)).toBe(true)
+            expect(users.length).toBe(4)
+            users.forEach((user) => {
+                expect(user).toMatchObject({
+                    username: expect.any(String),
+                    name: expect.any(String),
+                    avatar_url: expect.any(String)
+                })
+            })
+        })
+    })
+    test('404: responds with error message when URL is invalid', () => {
+        return request(app)
+        .get('/api/5')
+        .expect(404)
+        .then(( { body } ) => {
+            const errorMsg = body.msg
+            expect(errorMsg).toBe('404: Not Found')
+        })
```

Comment:
Because this error test is not going through /api/users then the test should live in it's own describe block. 

Also, if you have tested for an incorrect endpoint before that doesn't match, you don't need to do it again :)